src/css-raw/perf-cascade.css

```diff
@@ -48,14 +48,21 @@
 .row-item.status4xx .even,
 .row-item.status4xx .odd,
 .row-item.status3xx .even,
-.row-item.status3xx .odd {opacity: 0.3}
+.row-item.status3xx .odd {opacity: 0.3;}
 .row-item.status5xx:hover .even,
 .row-item.status5xx:hover .odd,
 .row-item.status4xx:hover .even,
 .row-item.status4xx:hover .odd,
 .row-item.status3xx:hover .even,
-.row-item.status3xx:hover .odd {opacity: 0.5}
+.row-item.status3xx:hover .odd {opacity: 0.5;}
 
+/** overlay animation settings **/
+.water-fall-chart .line-label-holder,
+.row-item,
+.water-fall-chart .line-holder line,
+.time-scale line,
+.time-scale text {transition:transform 60ms;}
+.water-fall-chart.closing {transition-delay: 60ms;}
```

Comment:
Delay to allow the closing animation to finish when closing - I decided not to animate the `water-fall-chart` height, since it'd force the whole page to be redraw as well.